beacon_node/beacon_chain/src/sync_committee_verification.rs

```diff
@@ -314,15 +324,22 @@ impl<T: BeaconChainTypes> VerifiedSyncContribution<T> {
             return Err(Error::AggregatorNotInCommittee { aggregator_index });
         };
 
-        // Ensure the valid sync contribution has not already been seen locally.
-        let contribution_root = contribution.tree_hash_root();
+        // Ensure the valid sync contribution or it's superset has not already been seen locally.
```

Comment:
Sorry to be "that guy" who corrects tiny typos 😅 

```suggestion
        // Ensure the valid sync contribution or its superset has not already been seen locally.
```

Reply:
Not at all, appreciate it :)